modules/end-user-guide/pages/selecting-a-workspace-ide.adoc

```diff
@@ -7,7 +7,7 @@
 [id="selecting-a-workspace-ide"]
 = Selecting a workspace IDE
 
-The default in-browser IDE in a new workspace is link:https://theia-ide.org/docs/[Che Theia]. In addition, link:https://github.com/che-incubator/che-code[Visual Studio Code] is available as a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] feature for non-restricted environments.
+The default in-browser IDE in a new workspace is link:https://github.com/che-incubator/che-code[Microsoft Visual Studio Code - Open Source]. In addition, link:https://github.com/che-incubator/jetbrains-editor-images[Community Editions of JetBrains IntelliJ IDEA and JetBrains PyCharm] are available as link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] features. link:https://theia-ide.org/docs/[Che-Theia] is deprecated and will be removed in the next release.
```

Comment:
Nick wants us to use "Eclipse Theia" according to https://issues.redhat.com/browse/CRW-3098, so I'm updating it here:
```suggestion
The default in-browser IDE in a new workspace is link:https://github.com/che-incubator/che-code[Microsoft Visual Studio Code - Open Source]. In addition, link:https://github.com/che-incubator/jetbrains-editor-images[Community Editions of JetBrains IntelliJ IDEA and JetBrains PyCharm] are available as link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] features. link:https://theia-ide.org/docs/[Eclipse Theia] is deprecated and will be removed in the next release.
```